nimare/meta/cbmr.py

```diff
@@ -110,10 +110,10 @@ def __init__(
         spline_spacing=10,
         model=models.PoissonEstimator,
         penalty=False,
-        n_iter=1000,
-        lr=1e-2,
+        n_iter=2000,
+        lr=1,
         lr_decay=0.999,
```

Comment:
what's the rationale for the changes to the defaults? is it because of the changes to the optimization procedure?

Reply:
the rationale is to take a full step for the initial iteration, then the learning rate decay will exponentially reduce the step size.